### PR TITLE
Strata - initial dashboard

### PR DESCRIPTION
**Pull request by deetz99**
Initial strata dashboard work.

More design/discussion will be needed before continuing.

### Changes
- rename pages
- add dashboard card and table

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the strr license (Apache 2.0).

### CODE REVIEW

File: strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue [hunk 1]
```diff
@@ -63,13 +68,13 @@ useHead({
 
 definePageMeta({
   layout: 'connect-dashboard',
-  middleware: ['auth'],
-  path: '/strata-hotel/dashboard/:registrationId'
```
Comment (kialj876) [inline strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue:67]: is the 'path' not necessary due to the page naming convention?

File: strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue [hunk 2]
```diff
@@ -1,60 +1,65 @@
...
+//     representatives.value = getDashboardRepresentives()
+//     // update breadcrumbs with platform business name
+//     // setBreadcrumbs([
+//     //   { label: t('label.bcregDash'), to: useRuntimeConfig().public.registryHomeURL + 'dashboard' },
+//     //   { label: strataBusiness.value.legalName }
+//     // ])
+//   }
+
+//   loading.value = false
+// })
```
Comment (kialj876) [inline strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue:62]: was this failing? I hadn't tested it very thoroughly but I thought all this commented out stuff was working

**Review by kialj876 — APPROVED** [review]
Looks good, I have a few comments below, but feel free to merge whenever you're ready

**Review by thorwolpert — APPROVED** [review]

File: strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue (hunk 1 above)
Comment (deetz99) [reply strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue:67]: Yeah the path will be created using the file structure, or you can override it with the definePageMeta 'path' property

File: strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue (hunk 2 above)
Comment (deetz99) [reply strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue:62]: Yeah, in the base layer, the todoItems util uses the platform store, which doesn't exist in the strata app

**Pull request merged**